azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/viewmodel/MessageViewModel.kt

```diff
@@ -24,28 +25,31 @@ internal class MessageViewModel(
     val dateHeaderText: String?,
     val isLocalUser: Boolean,
     val isRead: Boolean,
-
+    val isMaskedUser: Boolean,
```

Comment:
Masked to me suggests that we put a mask on the participant and show it "masked". But we do not show this participant at all. Should we call it isHiddenUser? or isVisible

Reply:
fixed